src/ipc/ipc-common.c

```diff
@@ -170,6 +170,11 @@ void ipc_send_queued_msg(void)
 	k_spin_unlock(&ipc->lock, key);
 }
 
+#ifdef __ZEPHYR__
+static struct k_work_q ipc_send_wq;
+static K_THREAD_STACK_DEFINE(ipc_send_wq_stack, 2048);
```

Comment:
CONFIG_IPC_SEND_STACK_SIZE - lets make this a tunable.

Reply:
@lgirdwood is it really worth it? This only occurs at one place. Maybe just put it in a header together with an additional one for https://github.com/thesofproject/sof/blob/c7b7c91ac71b5eaf32181adf67f4db97f4c080ec/zephyr/edf_schedule.c#L17

Reply:
Kconfig is cheap - we should have this option for all system threads.

Reply:
@lgirdwood thinking a bit more - maybe we can have a single SOF work queue, run on core 0, serving both these (and potentially other) needs? EDF is currently used by IPC reception, KPB, mtrace logging, we could move the work queue out of it to make it global and then also use it for IPC sending?

Reply:
We can look at this as a secondary phase, yes this will save memory but we need to look at what will/wont work here.

Reply:
@lgirdwood if that approach is acceptable, I'd rather do that than adding significant infrastructure to only remove it again soon. I'll push that version for a test.